src/nspawn/nspawn-expose-ports.c

```diff
@@ -106,19 +106,19 @@ int expose_port_flush(FirewallContext **fw_ctx, ExposePort* l, union in_addr_uni
                                       p->container_port,
                                       NULL);
                 if (r < 0)
-                        log_warning_errno(r, "Failed to modify firewall: %m");
+                        log_warning_errno(r, "Failed to modify %sfirewall: %m", af == AF_INET6 ? "ipv6 ": "");
```

Comment:
use `af_to_name(af)` here, include `af-list.h` for that

Reply:
much better, thanks!